nemo/collections/asr/parts/submodules/subsampling.py

```diff
@@ -251,19 +251,128 @@ def __init__(
                     )
                 layers.append(activation)
                 in_channels = conv_channels
+
+        elif subsampling == 'striding_conv1d_k5':
```

Comment:
I'm not sure about the _k5 in the name. Best to remove it and keep it just ..._conv1d.

Reply:
Agree with the same, pls remove _k5.

Reply:
done, I removed the _k5 in new commit

Reply:
thanks, update the change log accordingly.